Class/A-Review/exercise1.js

```diff
@@ -5,7 +5,13 @@ let ageOfReshma = 25;
 // "Reshma is the same age as me" if Reshma is your age
 // "Reshma is not my age" if Reshma is older or younger than you
 
+let myAge = 33
 
+if (ageOfReshma === myAge){
+    console.log("yes");
+}else {
+    console.log("No");
+}

```

Comment:
While the logic is correct. You never logged `Reshma is not my age` or `Reshma is the same age as me`. It's important to make sure you're reading the tasks carefully.

Reply:
For sure, thanks for your attention.